appetiser.py

```diff
@@ -50,14 +52,34 @@ def ping():
 def convert():
     json_data = flask.request.get_json()
     appetiser.logger.info('Processing request data: %s', json_data)
-    result = process(**extract_process_kwargs(json_data))
-    proto_reponse = {
-        **result,
-        **extract_response_items(json_data)
-    }
-    response = add_iiif_info_json(proto_reponse)
-    appetiser.logger.info('Response: %s', json_data)
-    return flask.jsonify(response)
+    try:
+        result = process(**extract_process_kwargs(json_data))
+        proto_reponse = {
+            **result,
+            **extract_response_items(json_data)
+        }
+        response = add_iiif_info_json(proto_reponse)
+        appetiser.logger.info('Response: %s', json_data)
+        return flask.jsonify(response)
+    except FileNotFoundError as fileError:
+        appetiser.logger.exception('Error: %s', fileError)
+        return flask.jsonify(
+            status='file not found',
+            message=str(fileError)
+        ), 400
+    except subprocess.CalledProcessError as subprocess_error:
+        appetiser.logger.exception('Error: %s', subprocess_error.stderr)
+        return flask.jsonify(
+            status='kakadu error',
+            message=str(subprocess_error.stderr)
```

Comment:
it would produce output like the following - 
```
{
    "message": "b'/bin/sh: 1: Syntax error: \"(\" unexpected\\n'",
    "status": "kakadu error"
}
```

The stack trace is also logged, but not sent through to the api